src/SprykerEco/Zed/Computop/Business/Payment/Handler/Saver/Complete/PayPalExpressCompleteResponseSaver.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Computop\Business\Payment\Handler\Saver\Complete;
+
+use Generated\Shared\Transfer\ComputopApiPayPalExpressCompleteResponseTransfer;
+use Generated\Shared\Transfer\ComputopPaymentComputopOrderItemCollectionTransfer;
+use Generated\Shared\Transfer\ComputopPaymentComputopTransfer;
+use Generated\Shared\Transfer\ComputopSalesOrderItemTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use Spryker\Zed\Kernel\Persistence\EntityManager\TransactionTrait;
+use SprykerEco\Zed\Computop\ComputopConfig;
+use SprykerEco\Zed\Computop\Persistence\ComputopEntityManagerInterface;
+use SprykerEco\Zed\Computop\Persistence\ComputopRepositoryInterface;
+
+class PayPalExpressCompleteResponseSaver implements PayPalExpressCompleteResponseSaverInterface
+{
+    use TransactionTrait;
+
+    /**
+     * @var \SprykerEco\Zed\Computop\ComputopConfig
+     */
+    protected $computopConfig;
+
+    /**
+     * @var \SprykerEco\Zed\Computop\Persistence\ComputopEntityManagerInterface
+     */
+    protected $computopEntityManager;
+
+    /**
+     * @var \SprykerEco\Zed\Computop\Persistence\ComputopRepositoryInterface
+     */
+    protected $computopRepository;
+
+    /**
+     * @param \SprykerEco\Zed\Computop\ComputopConfig $computopConfig
+     * @param \SprykerEco\Zed\Computop\Persistence\ComputopEntityManagerInterface $computopEntityManager
+     * @param \SprykerEco\Zed\Computop\Persistence\ComputopRepositoryInterface $computopRepository
+     */
+    public function __construct(
+        ComputopConfig $computopConfig,
+        ComputopEntityManagerInterface $computopEntityManager,
+        ComputopRepositoryInterface $computopRepository
+    ) {
+        $this->computopConfig = $computopConfig;
+        $this->computopEntityManager = $computopEntityManager;
+        $this->computopRepository = $computopRepository;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return \Generated\Shared\Transfer\QuoteTransfer
+     */
+    public function save(QuoteTransfer $quoteTransfer): QuoteTransfer
+    {
+        $payPalExpressCompleteResponseTransfer = $quoteTransfer->getPayment()
```

Comment:
I guess validation will complain on this. Each of those params also can be null

Reply:
fixed above

Reply:
but what about getComputopPayPalExpress?